pkg/controller/pod.go

```diff
@@ -440,11 +440,20 @@ func podContainerID(pod *corev1.Pod) string {
 }
 
 func addIfaceEventFormat(pod *corev1.Pod, network *nadv1.NetworkSelectionElement) string {
+	attributes := ""
+	if len(network.IPRequest) > 0 || network.MacRequest != "" || network.CNIArgs != nil {
+		attributes = fmt.Sprintf(" (ips: %v, mac: %s, cni-args: %v)",
```

Comment:
nit: I would remove the first space here, and add it in the sprintf below (in the return func).

Somehow I find that more readable.